geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/dunit/CqDataDUnitTest.java

```diff
@@ -957,22 +957,15 @@ public void run2() throws CacheException {
 
   /**
    * This test was created to test executeWithInitialResults being called multiple times.
-   * Previously, the queueEvents would be overwritten and we would lose data. This test will execute
-   * the method twice. The first time, the first execution will block it's own child thread (TC1).
-   * The second execution will block until TC1 is completed (based on how executeWithInitialResults
-   * is implemented) A third thread will be awaken and release the latch in the testhook for TC1 to
-   * complete.
+   * Previously, the queueEvents would be overwritten and we would lose data.
    *
    */
   @Test
```

Comment:
Test changes doesn't seem to be doing what its was originally intend to do. Previously with the test-hook; its making one of the thread to be waiting for "queuedEventsSynchObject" while other thread is doing Cq query execution.